docs/rails.md

```diff
@@ -37,6 +37,10 @@ Next, create a bootstrap `Gemfile` which just loads Rails. It'll be overwritten
     source 'https://rubygems.org'
     gem 'rails', '4.2.0'
 
+We'll also need an empty `Gemfile.lock` in order to build our `Dockerfile`.
```

Comment:
You'll need an empty `Gemfile.lock` file to build the `Dockerfile`.

![image](https://cloud.githubusercontent.com/assets/1347209/10851383/09a9dfe2-7ee8-11e5-86df-03fe57bd98d5.png)